server/gitrest/packages/gitrest-base/src/utils/definitions.ts

```diff
@@ -33,7 +33,7 @@ export interface IRepositoryManager {
     createCommit(commit: git.ICreateCommitParams): Promise<git.ICommit>;
     getRefs(): Promise<git.IRef[]>;
     getRef(ref: string, externalWriterConfig?: IExternalWriterConfig): Promise<git.IRef>;
-    createRef(createRefParams: git.ICreateRefParams, externalWriterConfig?: IExternalWriterConfig): Promise<git.IRef>;
+    createRef(createRefParams: git.ICreateRefParams & { force?: boolean }, externalWriterConfig?: IExternalWriterConfig): Promise<git.IRef>;
```

Comment:
would this become a breaking change if we directly added `force` as an optional parameter within `git.ICreateRefParams`?

Reply:
`git.ICreateRefParams` is from `@fluidframework/gitresources`. I don't think adding it as an optional param there would break anything, but it would require an r11s package release cycle, and I only want the force param specifically in Gitrest.